common/djangoapps/third_party_auth/settings.py

```diff
@@ -84,3 +85,15 @@ def apply_settings(django_settings):
         'social.apps.django_app.context_processors.backends',
         'social.apps.django_app.context_processors.login_redirect',
     )
+
+    set_strategy_getter()
+
+
+def set_strategy_getter():
+    """
+    Set strategy loader method to workaround current strategy getter needed on
+    get_user() method on authentication backends.
+    """
+    from social.strategies.utils import set_current_strategy_getter
+    from social.apps.django_app.utils import load_strategy
+    set_current_strategy_getter(load_strategy)
```

Comment:
My question would be, why does python-social-auth restrict that workaround to django 1.7 and below? It seems like they think it should work in django 1.8 without that code. Can you try and track down how python-social-auth is expecting to work in django 1.8?